keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1035,41 +819,15 @@ Some of the race conditions that need to be handled are:
   being deallocated. Because that is visible in the claim status, scheduling
   of the second pod cannot proceed.
 
-### Custom parameters
-
-To support arbitrarily complex parameters, both ResourceClass and ResourceClaim
-contain one field which references a separate object. The reference contains
-API group, kind and name and thus is sufficient for generic clients to
-retrieve the parameters. For ResourceClass, that object must be
-cluster-scoped. For ResourceClaim, it must be in the same namespace as the
-ResourceClaim and thus the Pod. Which kind of objects a resource driver accepts as parameters depends on
-the driver.
-
-This approach was chosen because then validation of the parameters can be done
-with a CRD and that validation will work regardless of where the parameters
-are needed.
-
-It is the responsibility of the resource driver to convert these CRD parameters
-into in-tree ResourceClaimParameters and ResourceClassParameters. Kubernetes
-finds those generated parameters based on their `generatedFrom` back-reference.
-
-Parameters may get deleted before the ResourceClaim or ResourceClass that
-references them. In that case, a pending resource cannot be allocated until the
-parameters get recreated. An allocated resource must remain usable and
-deallocating it must be possible. To support this, resource drivers must copy
-all relevant information:
-- For usage, the `claim.status.allocation.resourceHandle` can be hold some copied information
-  because the ResourceClaim and thus this field must exist.
-- For deallocation, drivers should use some other location to handle
-  cases where a user force-deletes a ResourceClaim or the entire
-  cluster gets removed.
-
 ### Sharing a single ResourceClaim
 
 Pods reference resource claims in a new `pod.spec.resourceClaims` list. Each
 resource in that list can then be made available to one or more containers in
-that Pod. Depending on the capabilities defined in the
-`claim.status.allocation` by the driver, a ResourceClaim can be used exclusively
+that Pod. Contains can also get access to a specific requested devices when the
+claims asked for more than one.
+
+Depending on the capabilities defined in the
+`claim.status.allocation`, a ResourceClaim can be used exclusively
 by one pod at a time or an unlimited number of pods. Support for additional
 constraints (maximum number of pods, maximum number of nodes) could be
 added once there are use cases for those.
```

Comment:
Is this based on the `Sharable` field (which it sounds like we may be dropping support for)?

Reply:
Removed the part about "Depending on the capabilities.." because now all claims are shareable.